features/set-methods.yml.dist

```diff
@@ -2,11 +2,14 @@
 # Do not edit this file by hand. Edit the source file instead!
 
 status:
-  baseline: false
+  baseline: low
+  baseline_low_date: 2024-06-11
   support:
     chrome: "122"
     chrome_android: "122"
     edge: "122"
+    firefox: "127"
+    firefox_android: "127"
```

Comment:
This actually showed up in BCD a couple of weeks ago (https://github.com/mdn/browser-compat-data/pull/23198), but only now appears in our data because the release has passed. Dealing with https://github.com/web-platform-dx/web-features/issues/955 would make this easier to observe at the time the data changed.